### PR TITLE
Grad/speaking

### PR DESCRIPTION
**Pull request by yipingbenjaminliu**
To allow assignment of trainees speaking
speaking trainees input portion
produces report of their speaking

### CODE REVIEW

**Comment by iamsitting** [comment]
It looks good to me. But one more review would be better @bdou

File: ap/graduation/templates/graduation/outline_report.html [hunk 1]
```diff
@@ -65,17 +65,17 @@ <h1>{{title}}</h1>
...
     <tbody>
       {% for c in data %}
       <tr>
         <td>{{c.trainee.firstname}}</td>
         <td>{{c.trainee.lastname}}</td>
-        <td>{{c.sections}}</td>
+        <td>{{c.speaking_section}}</td>
         <td>{{c.participate}}</td>
-        <td>{{c.sentence}}</td>        
+        <td>{{c.speaking}}</td>
```
Comment (bdou) [inline ap/graduation/templates/graduation/outline_report.html]: Maybe I misunderstand, but why is the outline report being changed also to basically show the same thing as speaking report? Why wouldn't it stay the same, while the new fields show up under the speaking report?

Comment (bdou) [reply ap/graduation/templates/graduation/outline_report.html]: other than that everything looks fine.

Comment (yipingbenjaminliu) [reply ap/graduation/templates/graduation/outline_report.html]: oh, whoops. Good catch, the outline-report shouldn't have been touched

Comment (iamsitting) [reply ap/graduation/templates/graduation/outline_report.html]: Good catch. @yipingbenjaminliu

**Comment by iamsitting** [comment]
Please use this: https://github.com/SublimeText/TrailingSpaces to address whitespace on template files.

**Pull request merged**